db/migrate/20240625234938_add_merchandise_to_race_editions.rb

```diff
@@ -0,0 +1,8 @@
+class AddMerchandiseToRaceEditions < ActiveRecord::Migration[7.0]
+  def change
+    add_column :race_editions, :selling_merchandise, :boolean, default: false
+    add_column :race_editions, :merchandise_description, :string, null: true
+    add_column :race_editions, :merchandise_image_file_name, :string, null: true
```

Comment:
Do we want to use the Rails built-in attachments feature for images? It's pretty nice.

Reply:
Sure. If there is a better/more Rails way to do it, let's do it! Can you point me to some info on this approach?

Reply:
Actually, let's wait on that. It sounds like a cleaner way to go -- creating a new ActiveRecord for merchandise, but without a separate record for the merchandise doesn't seem like using the attachments to the RaceEdition is best... but what do I know? I just looked for about 5 minutes. :-) We can always clean this up later.